### ISSUE TITLE
 [TSDoc> Playground]: There is no visual page heading present on the page as h1 which describes the page information.

### ISSUE THREAD

**Issue by vagpt** [microsoft/rushstack-websites#253]
**Test Environment:**
Version: Version 24H2 (OS Build 26231.5000)
Browser Version: 127.0.2638.2 (Official build) dev (64-bit)
URL: https://tsdoc.org/play/
Screen Reader: Narrator

**Repro Steps:**
1. Open the above URL on the EDGE Dev browser.
2. Press the tab key and navigate to the 'PlayGround' option and press enter key to activate it.
3. Navigate the page and observe the issue.

**Actual Result:**
There is no visual page heading is present on the page as h1 which describe the page information.

**Expected Result:**
There should be a descriptive page heading should be defined as H1.

**Note:**
It should be same as it is on 'Intro, Tags, Github and Help' page.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/headings-and-labels

Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment.

![There is no visually page description label is defined on the page which should be defined as H1](https://github.com/microsoft/rushstack-websites/assets/95913748/bb028249-88ec-4e5e-bf54-a7093f2430c3)
